src/SDKs/Network/Management.Network/Microsoft.Azure.Management.Network.csproj

```diff
@@ -7,7 +7,7 @@
 		<PackageId>Microsoft.Azure.Management.Network</PackageId>
 		<Description>Provides management capabilities for Network services.</Description>
 		<AssemblyName>Microsoft.Azure.Management.Network</AssemblyName>
-		<Version>18.0.1-preview</Version>
+		<Version>19.0.1-preview</Version>
```

Comment:
1.  Versioning should start from  19.0.0-preview
2. `PackageReleaseNotes` must be updated here

@shahabhijeet Doesn't look like this PR has any breaking changes but the spec from which this code was generated was from a new version, should we still mark this as a major package revision?